addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1428,23 +1428,29 @@ export class OdooEditor extends EventTarget {
         const block = closestBlock(sel.anchorNode);
         for (const [style, tag, isList] of [
             ['paragraph', 'P', false],
+            ['pre', 'PRE', false],
             ['heading1', 'H1', false],
             ['heading2', 'H2', false],
             ['heading3', 'H3', false],
+            ['heading4', 'H4', false],
+            ['heading5', 'H5', false],
+            ['heading6', 'H6', false],
```

Comment:
Curiosity: what's this? I see we already have button for h4 -> h6 so I wonder... :grimacing:

Reply:
mmm... unless I'm missing something this defines the list of elements to be checked to toggle the `.active` class.
These `h[x]` were missing... I'll check with the editor team